azure-pipelines.yml

```diff
@@ -361,6 +372,54 @@ jobs:
 
     displayName: 'Test pip Install'
 
+- job: UnitTest
+  displayName: Unit Test for Core and Telemetry
+  timeoutInMinutes: 10
+
+  pool:
+    vmImage: 'ubuntu-16.04'
+  strategy:
+    matrix:
+      Python27:
+        python.version: '2.7'
+      Python36:
+        python.version: '3.6'
+      Python38:
+        python.version: '3.8'
+  steps:
+    - task: UsePythonVersion@0
+      displayName: 'Use Python $(python.version)'
+      inputs:
+        versionSpec: '$(python.version)'
+    - bash: pip install --upgrade pip tox
+      displayName: 'Install pip and tox'
+    - bash: ./scripts/ci/unittest.sh
+      displayName: 'Run Unit Test'
+
+- job: IntegrationTestAgainstProfiles
+  displayName: Integration Test against Profiles
+  dependsOn: BuildPythonWheel
+  condition: succeeded()
+  timeoutInMinutes: 20
+
+  pool:
+    vmImage: 'ubuntu-16.04'
+  strategy:
+    matrix:
```

Comment:
Do we need 2.7 too?

Reply:
Python 2.7 will get retired by 2020.01

Reply:
For awareness, there's a PR scheduled in S163 to drop Python 2.7 support: https://github.com/Azure/azure-cli/pull/11363.

Reply:
Let's Vote!

Reply:
Python 2

Reply:
Python 3

Reply:
question, python 2.7 retires at 2020.01, what's our plan on python 2.7 support? this may not related with PR directly.

Reply:
We're going to drop support for python 2.7 in the next sprint. The plan is to remove tests for python 2.7 and also remove python 2.7 from the classifiers in setup.py. So future code is not guaranteed to be compatible with python 2.7. 

We will not have active support for python 2.7, but anyone from the community could submit PRs to fix python-2.7-breaking code as long as the impact is small.